internal/worker/json.go

```diff
@@ -217,6 +217,26 @@ type DepsolveJobResult struct {
 	JobResult
 }
 
+// SearchJob defines the parameters for a dnf metadata search
+// It will search the included repositories for packages matching the
+// package strings
+// TODO -- Does this support globs of any kind?
+type SearchJob struct {
```

Comment:
Nitpick for both `SearchJob` and `SearchJobResult`: I suggest including "package" in the struct name to clarify the job purpose. While we will probably be ever depsolving only packages, we may end up searching for more things just for packages.

Reply:
Let's see what folks think.